test/operator-framework-e2e/setup.sh

```diff
@@ -18,31 +18,35 @@ The following environment variables are required for configuring this script:
 setup.sh also takes 5 arguments. 
 
 Usage:
-  setup.sh [OPERATOR_SDK] [CONTAINER_RUNTIME] [KUSTOMIZE] [KIND] [KIND_CLUSTER_NAME]
+  setup.sh [OPERATOR_SDK] [CONTAINER_RUNTIME] [KUSTOMIZE] [KIND] [KIND_CLUSTER_NAME] [NAMESPACE]
```

Comment:
Ah, does look like I forgot to increase the argument count there. I think it could be helpful to check for a non-empty namespace but I think requiring it to be an argument alleviates the need for that. You'd have to explicitly make it an empty value